.github/workflows/build_samples.yml

```diff
@@ -16,7 +16,7 @@ env:
     app/samples/shield_blinky
     app/samples/shield_mcp3561r
     app/samples/lora_bcast
-    app/samples/udb_bcast
```

Comment:
Can you add the fuse 32 bit library setup to build_samples as well?

Reply:
no